lib/tictactoe/cli/console.ex

```diff
@@ -0,0 +1,50 @@
+defmodule TicTacToe.CLI.Console do
+  def display_instructions do
+    IO.puts("Welcome to Tic Tac Toe!\n")
+    IO.puts(instructions())
+    IO.puts("Starting new game...\n")
+  end
+
+  def display_board(board) do
+    length = row_length(board.cells)
+
+    board.cells
```

Comment:
What do you think about pattern matching to get `cells` within the function signature?

Reply:
Would I do that like this? 
`  def display_board(%{cells: cells}) do
    length = row_length(cells)

    cells
    |> Enum.chunk_every(length)
    |> Enum.map_join(horizontal_line(length), &format_row(&1))
    |> IO.puts()
  end`

Reply:
You've got the right idea. However, this will match on **any** struct or map that contains a `cells` attribute. How can we make this pattern match "more strict" by only matching on `Board` structs?

Reply:
So alias TicTacToe.Board into this file and then:
```
  def display_board(%Board{cells: cells}) do
    length = row_length(cells)
    ```